.nuxtrc

```diff
@@ -1 +1,2 @@
 typescript.includeWorkspace=true
+imports.autoImport=false
```

Comment:
The project has `.editorconfig` which explicitly forces final new line:

https://github.com/enkot/nuxt-open-fetch/blob/55a619189adde50a418c64d1d1da1bd93c0bdc9c/.editorconfig#L9

Please lint your code accordingly.

Reply:
Done